pkg/driver/node_test.go

```diff
@@ -692,3 +753,18 @@ func getNodeMock(mockCtl *gomock.Controller, nodeName string, returnNode *corev1
 
 	return mockClient, mockNode
 }
+
+func expectErr(t *testing.T, actualErr error, expectedCode codes.Code) {
+	if actualErr == nil {
+		t.Fatalf("Expect error but got no error")
```

Comment:
Nit: could be a touch more descriptive here
`t.Fatalf("Expect error code %d, but got no error", expectedCode)`